eo-maven-plugin/src/test/java/org/eolang/maven/optimization/OptCachedTest.java

```diff
@@ -199,9 +205,26 @@ private static XML program(final ZonedDateTime time, final String something) {
                     .add("program")
                     .attr("name", "main")
                     .attr("time", time.format(DateTimeFormatter.ISO_INSTANT))
+                    .add("objects")
                     .attr("something", something)
                     .up()
             ).xmlQuietly()
         );
     }
+
+    /**
+     * Adds attribute "hash" in EO program for tests.
+     * @param xml XML.
+     * @return XML representation of program.
+     */
+    private static XML updatedProgram(final XML xml)
```

Comment:
@Yanich96 Maybe it's better to leave just `program(final XML xml)`? 
https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html